GliaWidgets/Localization2/Localization2.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+struct Localization2 {
+    let callVisualizer: CallVisualizer
+}
+
+extension Localization2 {
```

Comment:
This part seems like out of place considering the rest of the file's content.

Reply:
fixed [7ebe6c3](https://github.com/salemove/ios-sdk-widgets/pull/877/commits/7ebe6c35b8bd118e7ad5dd256847be668c74df43)